spring-components/rdf4j-spring-demo/src/main/java/org/eclipse/rdf4j/spring/demo/model/EX.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * ******************************************************************************
+ *  * Copyright (c) 2021 Eclipse RDF4J contributors.
+ *  * All rights reserved. This program and the accompanying materials
+ *  * are made available under the terms of the Eclipse Distribution License v1.0
+ *  * which accompanies this distribution, and is available at
+ *  * http://www.eclipse.org/org/documents/edl-v10.php.
+ *  ******************************************************************************
+ */
+
+package org.eclipse.rdf4j.spring.demo.model;
+
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+
+/**
+ * @since 4.0.0
+ * @author Florian Kleedorfer
+ */
+public class EX {
+	private static final String base = "http://example.org/";
+	public static final IRI Artist = SimpleValueFactory.getInstance().createIRI(base, "Artist");
```

Comment:
Could we replace this with `Values.iri` instead? It's what we're hoping to promote as the recommended pattern these days.

Reply:
I'll look into it. No reservations whatsoever.